lib/client/auth-code.js

```diff
@@ -37,6 +37,8 @@ module.exports = function (config) {
    *                             An error object is passed as first argument and the result as last.
    */
   function getToken(params, callback) {
+    params.client_id = config.clientID;
+    params.client_secret = config.clientSecret;
```

Comment:
I was having the same issue and used the same fix.  Any reason this hasn't been merged with Master yet?